client/src/components/Logout/index.js

```diff
@@ -1,22 +1,30 @@
-import React from 'react';
+import React, { Component } from 'react';
 import axios from 'axios';
 
-const Logout = (props) => {
-    axios
+class Logout extends Component {
+    componentWillMount() {
+        axios
         .get(`/api/logout`)
-        .then(response => {
```

Comment:
A good example of how to do the code more testable!!! :D 
I remember how much problems I had trying to achieve 100% coverage with the old code... :D